raiden/transfer/mediated_transfer/target.py

```diff
@@ -236,7 +259,22 @@ def handle_block(target_state, channel_state, block_number):
         transfer.lock.secrethash,
     )
 
-    if not secret_known and block_number > transfer.lock.expiration:
+    secrethash = transfer.lock.secrethash
+    locked_lock = channel_state.our_state.secrethashes_to_lockedlocks.get(secrethash)
```

Comment:
The target is the node receiving a payment, not sending it. There will never be a lock on our state.

Please remove this logic.

Reply:
Target will receive the `LockExpired` message in this, but shouldn't it also handle cleaning up expired locks anyway? If so, then i changed the code to check the partner_state as that's where the check should happen.

Reply:
Done